sdk/dotnet/Pulumi.Automation.Tests/LocalWorkspaceTests.cs

```diff
@@ -826,5 +827,34 @@ static void AssertUpResult(UpResult upResult, string value)
                 Assert.True(expSecretValue.IsSecret);
             }
         }
+    
+        [Fact]
+        public async Task PulumiVersionTest()
+        {
+            using var workspace = await LocalWorkspace.CreateAsync();
+            Assert.Matches("(\\d+\\.)(\\d+\\.)(\\d+)(-.*)?", workspace.PulumiVersion.ToString());
+        }
+
+        [Theory]
+        [InlineData("100.0.0", true)]
+        [InlineData("1.0.0", true)]
```

Comment:
Isn't 2.x good enough when the minimal requirement is 1.0.0?

Reply:
No. Because a major version bump can include a breaking change to the CLI we will require major version parity for Automation SDK.